Memeolist/Memeolist/GeneratedAPI.swift

```diff
@@ -2,62 +2,18 @@
 
 import Apollo
```

Comment:
Let's add this to gitignore. This is generated and it should not be pushed back to the git.